muffnn/mlp/base.py

```diff
@@ -252,6 +259,10 @@ def _set_up_graph(self):
                 if self.keep_prob != 1.0:
                     t = tf.nn.dropout(t, keep_prob=self._keep_prob)
                 t = affine(t, layer_sz, scope='layer_%d' % i)
+
+            if self._transform_layer == i:
```

Comment:
could you just use `transform_layer_index` here and avoid setting `self._transform_layer` above?

Reply:
@beckermr suggested we do it this way

Reply:
why?

Reply:
could you have it be a property?